### PR TITLE
migrate remaining entities to getXXXOrFail lookups

### PR DESCRIPTION
**Pull request by techsmyth**

### CODE REVIEW

File: src/domain/actor/actor.service.ts [hunk 1]
```diff
@@ -36,23 +36,24 @@ export class ActorService {
...
     await this.actorRepository.delete(actorID);
     return true;
   }
 
   async updateActor(actorID: number, actorData: ActorInput): Promise<IActor> {
-    const actor = await this.getActor(actorID);
+    const actor = await this.getActorOrFail(actorID);
     if (!actor)
       throw new EntityNotFoundException(
         `Not able to locate actor with the specified ID: ${actorID}`,
```
Comment (atstoyanov) [inline src/domain/actor/actor.service.ts]: `getActorOrFail` is already checking for entity existance

File: src/domain/aspect/aspect.service.ts [hunk 2]
```diff
@@ -27,30 +27,26 @@ export class AspectService {
...
     if (!aspect)
       throw new EntityNotFoundException(
         `Not able to locate aspect with the specified ID: ${aspectID}`,
         LogContext.CHALLENGES
       );
-    await this.aspectRepository.delete(aspectID);
-    return true;
-  }
-
-  async getAspect(aspectID: number): Promise<IAspect | undefined> {
```
Comment (atstoyanov) [inline src/domain/aspect/aspect.service.ts:40]: Why this is removed?

File: src/domain/actor/actor.service.ts [hunk 3]
```diff
@@ -36,23 +36,24 @@ export class ActorService {
     return actor;
   }
 
-  async getActor(actorID: number): Promise<IActor | undefined> {
```
Comment (atstoyanov) [inline src/domain/actor/actor.service.ts:39]: Why this is removed?

File: src/domain/relation/relation.service.ts [hunk 4]
```diff
@@ -58,17 +58,19 @@ export class RelationService {
     return true;
   }
 
-  async getRelation(relationID: number): Promise<IRelation | undefined> {
```
Comment (atstoyanov) [inline src/domain/relation/relation.service.ts:61]: Why this is removed?

File: src/domain/tagset/tagset.service.ts [hunk 5]
```diff
@@ -40,18 +40,19 @@ export class TagsetService {
     return tagset;
   }
 
-  async getTagset(tagsetID: number): Promise<ITagset | undefined> {
```
Comment (atstoyanov) [inline src/domain/tagset/tagset.service.ts:43]: Why this is removed?

File: src/domain/user/user.service.ts [hunk 6]
```diff
@@ -69,20 +65,26 @@ export class UserService {
...
+      const idInt: number = parseInt(userID);
+      return await this.getUserByIdOrFail(idInt);
     }
 
-    const user = await this.getUserByEmail(userID);
-    if (user) return user;
+    return await this.getUserByEmailOrFail(userID);
   }
 
-  async getUserByID(userID: number): Promise<IUser | undefined> {
```
Comment (atstoyanov) [inline src/domain/user/user.service.ts:84]: Why this is removed?

**Review by atstojanov — COMMENTED** [review]
Have comments

File: src/domain/actor/actor.service.ts (hunk 3 above)
Comment (techsmyth) [reply src/domain/actor/actor.service.ts:39]: replaced by getActorOrFail

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**